mParticle-ComScore.podspec

```diff
@@ -16,7 +16,7 @@ Pod::Spec.new do |s|
     s.ios.deployment_target = "8.0"
     s.ios.source_files      = 'mParticle-ComScore/*.{h,m,mm}'
     s.ios.dependency 'mParticle-Apple-SDK/mParticle', '~> 6.12.0'
```

Comment:
I would prefer to let the minor version fluctuate, since it gives us flexibility in the release of bug fixes.